guides/common/modules/con_migrating-project-to-el8.adoc

```diff
@@ -1,9 +1,9 @@
 [id="migrating-project-to-el8_{context}"]
 ifdef::satellite[]
-= Migrating {ProjectName} to {RHEL} 8
+= Migrating {Project} to {RHEL} 8
 endif::[]
 ifndef::satellite[]
-= Migrating {ProjectName} to Enterprise Linux 8
+= Migrating {Project} to Enterprise Linux 8
```

Comment:
Should this read "Migrating Project to a new Enterprise Linux system", to emphasize that you're moving to a different machine?